airbyte-api/src/main/openapi/config.yaml

```diff
@@ -995,6 +997,38 @@ paths:
                 $ref: "#/components/schemas/HealthCheckRead"
         "404":
           description: Debug info not found
+  /v1/migration/export:
+    post:
+      tags:
+        - migration
+      summary: Export Airbyte Configuration and Data
+      operationId: exportAirbyte
```

Comment:
we should come up with a name for what the bundled data is called. I think you had previously used the word `archive`. I thought that was catchy? Should we use that? 

If so then `exportAirbyte` => `exportArchive`, `importAirbyte` => `importArchive`